woocommerce/woocommerce-android
# [Wear App] Handle application password stores scenario

Summary
==========
Fix issue #12132, where authentication with an Application Password in the Wear app causes a crash. The fix, for now, consists of disabling the direct wear-to-store connection through FluxC when the Store is not connected through a bearer token and using the phone connection instead.

How to Test
==========
1. In the mobile app, authenticate with using an Application Password method through the Store credentials screen.
2. Open the Wear app, make sure the sync starts, the store is loaded and all features are working as expected (Store stats, Orders list and Order details).

Update release notes:
